doc/releases/changelog-0.41.0.md

```diff
@@ -420,29 +420,40 @@ With `qml.decompositions.enable_graph()`, the following new features are availab
 
 <h4>QROM State Preparation 📖</h4>
 
-* Added template `qml.QROMStatePreparation` that prepares arbitrary states using `qml.QROM`.
+Quantum Read-Only Memory (QROM) isn't just useful for loading bitstrings.
+It can be leveraged to prepare entire quantum states!
+
+* Added template :class:`qml.QROMStatePreparation <pennylane.QROMStatePreparation>` that prepares arbitrary states using `qml.QROM`.
```

Comment:
I suggest putting the immediate impact on the user closer to the start of the changelog entry 💪. E.g., 

```suggestion
* A new state-of-the-art state preparation technique based on QROM is now available with the :class:`qml.QROMStatePreparation <pennylane.QROMStatePreparation>` template.
```